### PR TITLE
Changes for better matching of TV show releases.

### PR DESCRIPTION
**Pull request by AndreasStokholm**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -8,13 +8,13 @@ module.exports = function(name){
     year: match(name, /(?!^)[1,2]\d{3}/),
     resolution: match(name, /\d{3,4}p/i),
     type: match(name, /CAM|TS|TELESYNC|(DVD|BD)SCR|SCR|DDC|R5\.LINE|R5|DVDR|(HD|PD)TV|WEB-DL|WEBDL|BluRay|(DVD|HD|BR|BD)Rip/i),
-    video: match(name, /[xh]\.?264/i),
-    audio: match(name, /AAC2\.0|AAC|AC3|DTS|DD5\.1/i),
+    video: match(name, /[xh]\.?264|[xh] \.?264/i),
+    audio: match(name, /AAC2\.0|AAC|AC3|DTS|DD5\.1|DD5 1/i),
```
Comment (matiassingers) [inline index.js]: How about using the whitespace character and make it something like this:

``` js
video: match(name, /[xh][\.\s]?264/i),
audio: match(name, /AAC2[\.\s]0|AAC|AC3|DTS|DD5[\.\s]1/i),
```

Then later maybe pull out the `[\.\s]` into a setting or something.

File: index.js [hunk 2]
```diff
@@ -8,13 +8,13 @@ module.exports = function(name){
...
     language: match(name, /MULTiSUBS|MULTi|NORDiC|DANiSH|SWEDiSH|NORWEGiAN|GERMAN|iTALiAN|FRENCH|SPANiSH/i),
     edition: match(name, /UNRATED|DC|(Directors|EXTENDED)\.(CUT|EDITION)|EXTENDED|3D|2D/i),
     tags: name.match(/COMPLETE|LiMiTED|iNTERNAL/i),
     release: match(name, /PROPER|REPACK|READNFO|READ\.NFO|DiRFiX|NFOFiX/i),
-    group: match(name, /-[A-Za-z0-9]+$/).replace('-', '')
+    group: match(name, /[A-Za-z0-9]+$/)
```
Comment (matiassingers) [inline index.js:17]: Good point! :+1:

File: index.js (hunk 1 above)
Comment (AndreasStokholm) [inline index.js]: Yeah, that makes sense. I'll change it.

**Comment by matiassingers** [comment]
Awesome, can you please squash your commits and I'll merge it in?

**Comment by AndreasStokholm** [comment]
Yup. It's done.

**Comment by matiassingers** [comment]
Thanks! 

Published in [`v1.0.1`](https://github.com/matiassingers/scene-release/releases/tag/v1.0.1).

**Pull request merged**